test/e2e/conformance_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build conformance_e2e
```

Comment:
Is this something your editor is replacing?

Reply:
I'm not sure. I guess so? Is it an issue?